src/controllers/items.py

```diff
@@ -385,7 +458,8 @@ class SetTorrentRDResponse(BaseModel):
 def add_torrent(request: Request, id: int, magnet: str) -> SetTorrentRDResponse:
     torrent_id = ""
     try:
-        torrent_id = add_torrent_magnet(magnet)
+        _, infohash = get_type_and_infohash(magnet)
+        torrent_id = add_torrent(infohash)
```

Comment:
This will fail for AllDebrid :/

Reply:
I think that we should refactor some things related to the manual torrent adding when we merge #762 so we have a standard interface for the downloaders.

Reply:
I fully agree!

Reply:
This needs looked at again.